src/ui/components/SearchInput/index.js

```diff
@@ -30,32 +30,34 @@ export default class SearchInput extends React.Component {
 
   onBlur = () => {
     this.setState({ focus: false });
+    if (!this.value) {
+      // Animation will start if there is no value.
```

Comment:
Haha, end of the day Mark was trying, he really was! I guess if I'm writing comments on GitHub to explain something they just belong in the code 😛.